src/tests/ftest/pool/dynamic_server_pool.py

```diff
@@ -62,8 +62,7 @@ def check_pool_location(self, hosts, uuid_to_ranks):
             # appears that self.hostlist_servers[0] is always rank0, 1 is rank1,
             # and the extra server we'll be adding will be rank2.
             for rank, host in enumerate(hosts):
-                rc = check_for_pool(host, pool.uuid.lower())
-                pool_exists_on_host = rc == RC_SUCCESS
+                pool_exists_on_host = check_for_pool(host, pool.uuid.lower())
```

Comment:
We should also remove the instantiation of `RC_SUCCESS` as it's no longer used:
https://github.com/daos-stack/daos/blob/c1d11e9d74a059249eec29cc1be808c1e7ee6baa/src/tests/ftest/pool/dynamic_server_pool.py#L58

Reply:
Removed.